### DISCUSSION TITLE
JSON Input

### DISCUSSION THREAD

**Discussion by Designer023** [sanity-io/sanity#3813] [Sanity Studio Support]
Hi, 

Great work on V3. It's been really easy to migrate most of my site to use the new components with only a minor amount of issues.

One thing I am stuck on is a custom plugin. Previously in V2 I could use an input type of JSON in a schema/plugin (if I imported it):

```json
 {
      "implements": "part:@sanity/form-builder/input/json",
      "path": "JSONInput.js"
    },
```

Is this still possible?

**Comment by thisispete** [comment]
did you ever figure this out? having a hard time trying to get the sanity-plugin-json-input to work too

**Comment by bobinska-dev** [comment]
Hello everyone, you can define custom plugins like this in sanity.config.ts as a form component https://www.sanity.io/docs/form-components in general. 
You would need to follow this logic https://www.sanity.io/docs/installing-and-configuring-plugins
